### ISSUE TITLE
Test and update plugins with the release of the logstash-core gem.

### ISSUE THREAD

**Issue by purbon** [elasticsearch/logstash#2651]
Before moving forward with the release of the logstash-core gem, we should test and update accordingly all plugins that goes with logstash.

This is actually a blocker for 1.5.

Tasks:

- [x] easy the transition from the current way of doing things, to the new logstash-core gem one.
- [x] test installation and run to see everything works as expected.
- [x] publish a good  logstash-core gem for 1.5
- [x] Update the plugins used in testing, making sure they work with both systems (gemspec and logstash-core) 
- [x] test installation and run to see everything works as expected. (now with plugins using logstash-core)
- [x] prepare for the update of all plugins to using the logstash-core gem (including README and example plugins)
- [x] merge #2677 to master and 1.5 ( __taking care current plugins works still__ ).
- [x] release a logstash-core gem for 1.5.0.beta2, pending until we have a working newer version.
- [x] update all plugins to require logstash-core.
- [x] If everything works as expected, close this issue ;-)

This related to the pr #2677.

**Comment by purbon** [comment]
Testing has been done using an internal rubygems repo, but by releasing any gem public.

**Comment by purbon** [comment]
@colinsurprenant @ph will prepare the code to do the mass update so we can merge this soon, we should think how do we handle the fact that the currently released version for logstash-core-1.5.0-beta1 is actually boggus. 

We are force there to bump the minor version, so we lost the fact of having the same version as logstash.  ( as I said I incorrectly released there a gem without the locales, also need it for a proper execution)

**Comment by purbon** [comment]
Updates to the README.md are to remove this lines.

```
 
-The Logstash code required to run the tests/specs is specified in the `Gemfile` by the line similar to:
-```ruby
-gem "logstash", :github => "elasticsearch/logstash", :branch => "1.5"
-```
-To test against another version or a local Logstash, edit the `Gemfile` to specify an alternative location, for example:
-```ruby
-gem "logstash", :github => "elasticsearch/logstash", :ref => "master"
-```
-```ruby
-gem "logstash", :path => "/your/local/logstash"
-```
```
